docs/luigi-client-api.md

```diff
@@ -211,6 +238,141 @@ LuigiClient.linkManager().goBack({ foo: 'bar' });
 LuigiClient.linkManager().goBack(true);
 ```
 
+## splitView
+
+The Split View 
+
+-   Actions to control the split view
+-   Event handlers
+-   Get the current state
+
+### collapse
+
+Collapses the split view
+
+#### Examples
+
+```javascript
+splitViewHandle.collapse();
+```
+
+### expand
+
+Expands the split view
+
+#### Examples
+
+```javascript
+splitViewHandle.expand();
+```
+
+### setSize
+
+Sets the height of the split view
+
+#### Parameters
+
+-   `value` **[number](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Number)** lower height in percent
+
+#### Examples
+
+```javascript
+splitViewHandle.setSize(60);
+```
+
+### on
+
+Registers a listener for split view events
+
+#### Parameters
+
+-   `key` **enum** a set of predefined events: expand, collapse, resize, close
```

Comment:
Correct me if I am wrong, but the key is actually a string from an enum, not an enum itself

Reply:
done